extracting_assesment_of_feasibility.py

```diff
@@ -0,0 +1,76 @@
+""" This file is for getting the assesment of feasibility
+from xml and saving them to a csv file. """
+
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def extract_assesment_of_feasibility(url_df):
+    """Function to extract assesment of feasibility from the xml."""
+
+    # initialize lists
+    list_of_lists = []
+
+    # loop through the urls in the csv
+    for url in url_df:
+        # get project_id from url and save to the list
+        project_id = url.split("id=", 1)[1]
+        assesment_list = [project_id]
+
+        # get the url content
+        url = requests.get(url, timeout=5)
```

Comment:
put to top of file
DEFAULT_TIMEOUT = 5
and use it.
It will be easier to change default later